docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN conda config --append channels conda-forge
 RUN conda config --append channels bioconda
 
 RUN conda install python=3.10
-RUN conda install poppunk
+RUN conda install poppunk=2.6.5
```

Comment:
what was the reason to lock to this version? just wondering if there is updates to popunk we will need to update this every time

Reply:
it didn't seem to update to latest version for me, but I think maybe that was before the new version had the main label. 
Will try it again without.

Reply:
Actually, I think the issue was that the new version had only gone on bioconda, not conda forge. So I'm now specifying that channel, as I think that will always have the latest.